### ISSUE TITLE
content of section rather than page

### ISSUE THREAD

**Issue by dylan00j** [turnermm/annotate#3]
love the feature thanks
I would love to be able to have the pop up have a particular section of a page rather than a whole page if that was possible?
It would allow for quick creation of certain annotations rather writing the entire text or copying it.

Also, possible to include a link to the relevant page that has populated that annotation? SO that when mouse over shows worthwhile content of page, you could just click on it to visit - like Mediawiki does in wikipedia
thanks again
Dylan

**Comment by turnermm** [comment]
Interesting suggestions. Not sure tho when I will be able to look further.

**Comment by dylan00j** [comment]
thanks,
works OK as is though.
I just installed Bootstrap3 template theme. Annotate does not work properly with it. Any suggestions how I can modify so that it does?
thanks

**Comment by turnermm** [comment]
I'm afraid not.  Bootstrap3 makes modifications and I don't have the free time right now to look into it.  The bootstrap3 developer might be able to help you more quickly.

**Comment by Floffyko** [comment]
Yes, I agree too. The plugin is nice, quite flexible and intuitive but we have to find a way to make functional the references to sections, anchors or any other method to make the  link different sentences inside a single long page.  Nice Job till now. Thanks for your efforts and  good ideas :-)

**Comment by turnermm** [comment]
Not entirely sure what you want to do but perhaps the include plugin might help.

**Comment by Floffyko** [comment]
You are right, the include plugin is already doing this, your guessing is right, but there is no tooltip or popup in the equation. Lets then find a bridge between include and Annotate :-) (The work around for me will be to combine folded with include, but still, will not  feel like a genuine tooltip).

**Comment by turnermm** [comment]
I am no longer making updates to my plugins, except in the simplest of cases and bugs. You can check wikimatrix for possible assistance.
